x11-plugins/pidgin-sipe/pidgin-sipe-1.25.0-r4.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Pidgin Plug-in SIPE (Sip Exchange Protocol)"
+HOMEPAGE="https://sipe.sourceforge.io/"
+SRC_URI="https://downloads.sourceforge.net/sipe/${P}.tar.gz"
+
+inherit autotools
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="dbus debug kerberos ocs2005-message-hack openssl test voice"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/gmime:2.6
+	dev-libs/libxml2:=
+	net-im/pidgin[dbus?]
+	kerberos? ( virtual/krb5 )
+	openssl? ( dev-libs/openssl:= )
+	!openssl? ( dev-libs/nss )
```

Comment:
If it links against nss, it is very likely that's the case for nspr too (you could refer to the pkgconfig file of nss).